rspec_event_list.rb

```diff
@@ -0,0 +1,53 @@
+require 'rspec/autorun'
+require_relative 'event_list'
+
+describe EventList do
+
+  let(:event) {Event.new("Battle of the Bands", "Downtown", "2021-09-02", "16:00")}
+  let(:event_list) {EventList.new}
+
+  it "adds an event" do
+  	event_list.add_event("2021-09-02", event)
+  	puts('Test Case 1')
+  	p(event_list)
+  	puts('Test Case Ended')
```

Comment:
no need to print test case ended or print event list. Only thing that needs to be done in test cases is to use expect the result using expect keyword